build/config.js

```diff
@@ -0,0 +1,6 @@
+const config= {
+    "baseUrl": "https://o2r.uni-muenster.de/api/v1/",
+    "ercUrl": "http://localhost/erc/",
+    "title":" | o2r Demoserver",
```

Comment:
Can we switch the title dynamically to include the journal name? Would be a nice feature, not part of this PR obviously.

Reply:
issue created https://github.com/o2r-project/ojs-erc-plugin/issues/18